src/dlls/mscorpe/mscorpe/wrapper.cpp

```diff
@@ -60,6 +60,14 @@ HRESULT
 LoadMscorpe(HMODULE * phModule)
 {
```

Comment:
(I meant statically link the whole mscorpe - do not add a new .dll to the output.)